gtda/time_series/tests/test_embedding.py

```diff
@@ -72,7 +74,25 @@ def test_embedder_transform(parameters_type, expected):
     assert_almost_equal(embedder.fit_transform(signal), expected)
 
 
+def test_embedder_resample():
+    embedder = TakensEmbedding(time_delay=4, dimension=5, stride=3)
+    y_resampled = embedder.resample(y)
```

Comment:
There was a small misunderstanding -- here and elsewhere, I didn't mean to ask to skip the fitting *step*, just that the assignment to `embedder` was not necessary. I.e. I was advocating just writing `embedder.fit(signal)` on this line instead of `embedder = embedder.fit(signal)`. In fact, not calling `fit` *should* make this test fail because of the `check_is_fitted(self)` line https://github.com/giotto-ai/giotto-tda/blob/9319c01c06cfecc602ea3f0017e006d7ffe42ef6/gtda/time_series/embedding.py#L469. So I am now confused why the tests are passing...

Reply:
Update: The tests *did* fail -- https://dev.azure.com/maintainers/Giotto/_build/results?buildId=860&view=logs&jobId=c142dd40-08df-51f0-cc26-a7854a9e4782&j=c142dd40-08df-51f0-cc26-a7854a9e4782&t=5d5ccc5e-822a-5979-42c9-7d629af6aebc. See (#229).

Reply:
You are completely right and I know that there is the `is_fitted` check. I was sloppy and forgot to run the tests after the last modifications!